src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Windows/Properties/launchSettings.json

```diff
@@ -1,7 +0,0 @@
-{
-  "profiles": {
-	"MyExtensionsApp.Windows (Package)": {
```

Comment:
Why has this file and the pubxml files been removed. Please add them back

Reply:
they got git ignored when there were moved.